documentation/documentation.gradle.kts

```diff
@@ -1,4 +1,5 @@
 import org.asciidoctor.gradle.jvm.AbstractAsciidoctorTask
+import org.jetbrains.kotlin.fir.resolve.dfa.contracts.createArgumentsMapping
```

Comment:
:question: This import is unused, isn't it?

Reply:
Yeah. Just checked